app/src/main/java/com/kickstarter/viewmodels/MessagesViewModel.java

```diff
@@ -472,8 +485,8 @@ public ViewModel(final @NonNull Environment environment) {
     @Override public @NonNull Observable<List<Message>> messageList() {
       return this.messageList;
     }
-    @Override public @NonNull Observable<String> participantNameTextViewText() {
-      return this.participantNameTextViewText;
+    @Override public @NonNull Observable<String> creatorNameTextViewText() {
+      return this.creatorNameTextViewText;
```

Comment:
we don't need a version of this if the participant is not a creator? assume this was intentional but just checking because I can't remember what this will do in the UI.

Reply:
oooh this is actually a good call / edge case I don't think I tested, thanks for catching will audit

Reply:
ah--so this creator name text view text field is displayed under the project name up top, e.g.

![image](https://user-images.githubusercontent.com/4934046/30560827-1af58184-9c87-11e7-8839-5adbf10b5f82.png)

so since a message thread will always have a project as a context, the creator name displayed there will always be the creator's name regardless of participants